api/src/main/java/org/openmrs/module/stockmanagement/StockManagementActivator.java

```diff
@@ -34,8 +34,6 @@ public void started() {
 		try{
 			List<StartupTask> startUpTasks = new ArrayList<>();
 			startUpTasks.add(new LocationTagsSynchronize());
-			startUpTasks.add(new LocationTreeSynchronize());
```

Comment:
@makombe why would you remove these. Have you factored  that they are important to another distribution